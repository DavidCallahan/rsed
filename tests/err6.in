1
2
3 m0re
4
5 less
6
