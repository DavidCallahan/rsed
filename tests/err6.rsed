#!./rsed
print "Test 1"
skip to required "more"
copy all
