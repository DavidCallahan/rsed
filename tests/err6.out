Test 1
input 6: script 3: at end of input, failed required pattern: "more"
